client/my-sites/stats/post-trends/index.jsx

```diff
@@ -128,35 +127,47 @@ class PostTrends extends Component {
 	};
 
 	render() {
-		const { siteId, query } = this.props;
+		const { siteId, query, translate } = this.props;
 
-		const leftClass = classNames( 'post-trends__scroll-left', {
-			'is-active': this.state.canScrollLeft,
-		} );
+		// const leftClass = classNames( 'post-trends__scroll-left', {
+		// 	'is-active': this.state.canScrollLeft,
+		// } );
 
-		const rightClass = classNames( 'post-trends__scroll-right', {
-			'is-active': this.state.canScrollRight,
-		} );
+		// const rightClass = classNames( 'post-trends__scroll-right', {
+		// 	'is-active': this.state.canScrollRight,
+		// } );
 
 		/* eslint-disable jsx-a11y/click-events-have-key-events, wpcalypso/jsx-classname-namespace */
 		return (
 			<div className="post-trends">
 				{ siteId && <QuerySiteStats siteId={ siteId } statType="statsStreak" query={ query } /> }
-				<SectionHeader label={ this.props.translate( 'Posting activity' ) } />
-				<Card>
+
+				<div>
```

Comment:
Could we do without nesting the contents in an extra div?
```suggestion
```

Reply:
Done!